daemon/src/projection.rs

```diff
@@ -264,6 +264,8 @@ impl Cfd {
                 (CfdState::ContractSetup, vec![])
             }
             ContractSetupCompleted { dlc } => {
+                // Lock transaction has shared output, but it likely has change
+                // output too that can be highlighted
```

Comment:
But are you going to find the correct output based on the role? I am not sure that `dlc.script_pubkey_for` is correct here!

Reply:
I don't think this one is correct either, because the outputs the change outputs are added by the `bdk` wallet automatically. It may be hard to highlight these without introducing weird changes to `maia`.

Reply:
should we not bother with the lock transaction's change output in this PR then? I'm happy to create a separate issue that will track that, especially that it would need to interact with `maia`

Reply:
I would not bother. But maybe @thomaseizinger knows how to solve this quickly.

Reply:
We should be able to find the lock output based on the amount right? We know how much the parties locked up so that should be possible to easily expose as a function on `Dlc` then.

Reply:
Probably a good enough heuristic given how unlikely it is that both parties generate change outputs with the same amount.

Reply:
Created a small PR for `maia` to allow for that: https://github.com/comit-network/maia/pull/12

Reply:
> Created a small PR for `maia` to allow for that: [comit-network/maia#12](https://github.com/comit-network/maia/pull/12) - I will update this one one it gets merged

migrated the changes here.